### ISSUE TITLE
Installing Heroku on windows 10: Error installing package.

### ISSUE THREAD

**Issue by iam--nash** [heroku/heroku#1779]
after installing heroku-toolbelt on my system (windows 10).
It **keeps trying to install heroku core plugin**, in which I always got this error..
#### Error installing package.Try running again with GODE_DEBUG=info ####.

Another Question what does it mean running again with GODE_DEBUG=info?
**Newbie here** .

**Comment by dickeyxxx** [comment]
It's an environment variable. Run `set GODE_DEBUG=info`

**Comment by iam--nash** [comment]
@dickeyxxx 

After running *set GODE_DEBUG=info*
I am now getting this..
npm info using npm@2.14.7
npm info using node@v4.2.1
npm info attempt registry request try #1 at 9:50:00 PM
npm http request GET https://cli-npm.heroku.com/heroku-cli-addons
npm info attempt registry request try #1 at 9:50:00 PM
npm http request GET https://cli-npm.heroku.com/heroku-apps
npm info attempt registry request try #1 at 9:50:00 PM
npm http request GET https://cli-npm.heroku.com/heroku-fork
npm info attempt registry request try #1 at 9:50:00 PM
npm http request GET https://cli-npm.heroku.com/heroku-git
npm info attempt registry request try #1 at 9:50:00 PM
npm http request GET https://cli-npm.heroku.com/heroku-local
npm info attempt registry request try #1 at 9:50:00 PM
npm http request GET https://cli-npm.heroku.com/heroku-run
npm ERR! Windows_NT 10.0.10240

npm ERR! argv "C:\\User\\MyPC\\AppData\\Local\\heroku\\node-v4.2.1-windows-x86\\bin\node.exe" "C:\\Users\\MyPC\\AppData\\Local\\heroku\\node-v4.2.1-windows-x86\\lib\node_modules\\npm\\cli.js" "install" "heroku-cli-addons" " heroku-apps" "heroku-fork" "heroku-git" "heroku-local" "heroku-run" "heroku-status" "--loglevel=info"

npm ERR! node v4.2.1
npm ERR! npm v2.14.7

npm ERR! error:0906D06C:PEM routines:PEM_read_bio:no start line

**Comment by dickeyxxx** [comment]
Why does it have `C:\User` then `C:\Users`? Is one of your environment variables incorrect? What is the output of `set`?

**Comment by iam--nash** [comment]
There is no output when I tried to run set GODE_DEBUG=info in cmd.
I am not certain if i have an incorrect environment variables. but this is what's inside my path.

 **C:\Users\Fernandez\AppData\Roaming\npm**

**Comment by dickeyxxx** [comment]
You just showed me the extra output GODE_DEBUG=info displayed... does any of your env vars reference C:\User instead of C:\Users? That looks to be your problem.

**Comment by iam--nash** [comment]
@dickeyxxx 
the output that i posted a couple of hours ago was when i tried to run heroku command.
after running set GODE_DEBUG=info.
but before. i dont get that output .

**Comment by iam--nash** [comment]
upon checking my environment variable i don't see anything referencing to C:User .

**Comment by dickeyxxx** [comment]
That bit at the end `npm ERR! error:0906D06C:PEM routines:PEM_read_bio:no start line` looks suspicious. Are you behind a proxy?

**Comment by iam--nash** [comment]
nope, I am not under a proxy .and I always make sure to turn off windows defender when doing this things, for it might affect the request.

**Comment by iam--nash** [comment]
@dickeyxxx what is really happening when I run <code>set GODE_DEBUG=info</code> ? It doesn't output anything. but the output when i tried to run any heroku command changed.

**Comment by iam--nash** [comment]
@dickeyxxx after checking some other issues that might be related to my issue, I have found out on error #1543 .
That he has these following files which I dont have in my .heroku folder:
client folder
usage folder
autoupdate.last
error.log
heroku-cli.exe.

I only have the client folder .. what could have been my problem ? I haven't deleted any of those .

**Comment by Bigcountry425** [comment]
David bigcountry wardlaw, won't to say. You might want to check your infrastructure being you running with codes it maybe code is wrong or platform could be needed to be restarted. 
Between both they might not be lining up to gather or you might be able to shout it all down and let reboot together. and user  I would used (   users. ).thanks for letting me comment David bigcountry wardlaw
